aptos-move/framework/aptos-framework/sources/Genesis.move

```diff
@@ -55,6 +59,10 @@ module AptosFramework::Genesis {
             allow_validator_set_change,
             rewards_rate,
             rewards_rate_denominator,
+            increase_limit,
+            increase_limit_denominator,
+            withdrawal_limit,
+            withdrawal_limit_denominator,
         )
```

Comment:
we really need more than one init function or to pass in structs, this isn't really tenable :P

Reply:
Doesn't creating the struct require the same number of args anyway? We can split this into 2 init functions but it's also rather awkward as all of these parameters are related. Plus the first init function would need to instantiate the empty ValidatorSetConfigurations resource with 0 for everything.